keyboards/handwired/dactyl_manuform/6x7_fullrows/rules.mk

```diff
@@ -0,0 +1,24 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
```

Comment:
```suggestion
```

Reply:
For this, can I move it into my personal custom keymap, since it seems like this is needed to compile against my controllers? Or is there some other way to specify that? Or is this just not needed at all?

Reply:
This is already specified in `keyboards/handwired/dactyl_manuform/6x7_fullrows/info.json`.

Reply:
Oh, sorry, I did not realize those were equivalent, I will get that corrected, thank you.